launch/auton.launch

```diff
@@ -29,6 +29,15 @@
     ============
   -->
   <rosparam command="load" file="$(find mrover)/config/localization.yaml" />
+
+  <!-- global EKF -->
+  <arg name="ekf_start_delay" default="0" />
+  <node pkg="robot_localization" type="ekf_localization_node" name="global_ekf" clear_params="true" launch-prefix="bash -c 'sleep $(arg ekf_start_delay); $0 $@'" >
```

Comment:
what's the deal with this delay thing? When would we use it? If the default is zero should it be there at all?

Reply:
Check out #376, basically without this I was getting a bunch of junk values on sim startup that would completely throw off the EKF estimate. This is meant to be a temporary solution until that is addressed, and it is actually being used in auton_sim.launch

Reply:
hm, scary. Okay